graph-builder/Cargo.toml

```diff
@@ -18,3 +18,6 @@ serde_derive = "^1.0.70"
 serde_json = "^1.0.22"
 structopt = "^0.2.10"
 tar = "^0.4.16"
+dkregistry = { git = "https://github.com/camallo/dkregistry-rs.git", rev = "bacca3b7fb80edf7c38035d6f355672aa2536dd4"}
```

Comment:
Can you break this out into a separate commit?

Reply:
It can, I just need to put that commit first then, otherwise the code is broken. The commit containing this, and just this, will then have an unneeded dependency. Are we okay with having single commits in a PR which aren't consistent in itself?

Reply:
Of course. Commits within a PR are logically related.